WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/NavigationTarget.kt

```diff
@@ -31,4 +31,5 @@ sealed class NavigationTarget : Event() {
     data class ViewSearchTerms(val statsGranularity: StatsGranularity, val selectedDate: Date) : NavigationTarget()
     data class ViewAuthors(val statsGranularity: StatsGranularity, val selectedDate: Date) : NavigationTarget()
     data class ViewUrl(val url: String) : NavigationTarget()
+    class ViewMonthsAndYearsStats : NavigationTarget()
```

Comment:
We could make this and all other classes without parameters an `object`.

Reply:
When the target is an object, it might get consumed more than once and the LiveData will cause us to navigate to a screen we don't want to visit.